.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: bash
+
+# Use container-based infrastructure for quicker build start-up
+sudo: false
```

Comment:
Yes, see [the Travis docs](https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments) for details.  We don't need sudo access for this script execution.